tartiflette/resolver/resolver.py

```diff
@@ -33,7 +34,7 @@ def __init__(self, name: str, schema_name: str = "default"):
 
     def bake(self, schema):
         if not self._implementation:
-            raise Exception("No implementation given")
+            raise MissingImplementation("No implementation given")
```

Comment:
Can you add the resolver name in the message ? "No Implementation given for resolver < %s >" or something like that.